ui/lib/utils/tableColumns.tsx

```diff
@@ -60,10 +60,17 @@ function fieldsKeyColumn(transKeyPrefix: string): IColumn {
     minWidth: 150,
     maxWidth: 250,
     onRender: (rec) => {
-      if (rec.keyDisplay) {
-        return rec.keyDisplay
-      }
-      return <TransText transKey={`${transKeyPrefix}${rec.key}`} />
+      const indent = Array(rec.indentLevel || 0)
+        .fill('\t')
+        .join('')
+      return (
+        <>
+          <span style={{ whiteSpace: 'pre' }}>{indent}</span>
```

Comment:
maybe better to be `<span style={{ width: indentLevel * xxx }} /> `

Reply:
Good idea! will update it soon.

Reply:
Done, use padding-left instead of width to fill the space, width doesn't work for span unless we set the display to inline-block.